Wrong Operator for Metric measurement

PawelLiebich opened the pull request:
Change from "lt = less than" to "eq = equal to" for Operator in MetricsTrigger.

azurecla commented:
Hi __@trinadhk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (trinadhk). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
Product unit PM, trusted source. Please see email for instructions on using the private repository :)

PawelLiebich committed:
Wrong Operator for Metric measurement

Change from "lt = less than" to "eq = equal to" for Operator in MetricsTrigger.
